### DISCUSSION TITLE
Request to add the ntopng Package

### DISCUSSION THREAD

**Discussion by webmasterkai** [SmoothWAN/SmoothWAN#98] [Ideas and feature requests]
I'd like to see the addition of the [ntopng](https://www.ntop.org/products/traffic-analysis/ntop/) package. Ntopng is a network traffic monitoring tool that provides detailed network usage statistics and is widely used in network management and security. It provides a powerful web-based traffic monitoring, analysis and flow collection tool that can help with optimization, and security. It offers features such as network traffic capture, real-time traffic usage, and customizable traffic reports.

It has helped me many times to figure out what devices (software updates, file sync) are using all my mobile/LTE bandwidth.

**Comment by TalalMash** [comment]
Agreed, will be added next release.
Note (to-self) that the ntopng web UI will be slow and sluggish as the process will be de-prioritised in favour of network and VPN performance, since ntopng is pretty resource intensive.
Will be running chroot image or external disk. Image for Pi and x86 with the sufficient internal storage.
External on off-the-shelf router/AP combos will auto mount external storage flashed with balenaEtcher from [images found here](https://github.com/SmoothWAN/SmoothWAN-chroot-imagebuilder) and run ntopng based on the storage UUID with automount.
